i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/troubleshooting.md

```diff
@@ -20,73 +20,67 @@ title: Troubleshooting
  * See the License for the specific language governing permissions and
  * limitations under the License.
  -->
- 
-## Scheduler logs
 
-### Retrieve scheduler logs
+## 调度日志(Scheduler logs)
 
-Currently, the scheduler writes its logs to stdout/stderr, docker container handles the redirection of these logs to a
-local location on the underneath node, you can read more document [here](https://docs.docker.com/config/containers/logging/configure/).
-These logs can be retrieved by [kubectl logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs). Such as:
+### 检索调度日志
+
+调度器会将日志写入stdout/stderr，docker容器就会将这些日志重新导向到节点的本地位置，你可以从[这里](https://docs.docker.com/config/containers/logging/configure/)读到更多的文档，这些日志可以通过[kuberctl logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs)检索。如：
 
 ```shell script
-// get the scheduler pod
+//获取调度的pod
 kubectl get pod -l component=yunikorn-scheduler -n yunikorn
-NAME                                  READY   STATUS    RESTARTS   AGE
-yunikorn-scheduler-766d7d6cdd-44b82   2/2     Running   0          33h
+NAME READY STATUS RESTARTS AGE
+yunikorn-scheduler-766d7d6cdd-44b82 2/2 Running 0 33h
 
-// retrieve logs
+//检索日志
 kubectl logs yunikorn-scheduler-766d7d6cdd-44b82 yunikorn-scheduler-k8s -n yunikorn
 ```
 
-In most cases, this command cannot get all logs because the scheduler is rolling logs very fast. To retrieve more logs in
-the past, you will need to setup the [cluster level logging](https://kubernetes.io/docs/concepts/cluster-administration/logging/#cluster-level-logging-architectures).
-The recommended setup is to leverage [fluentd](https://www.fluentd.org/) to collect and persistent logs on an external storage, e.g s3. 
+在大多数的情况下，这个命令没有办法获取所有的日志，因为调度程序的日志数量庞大，您需要为集群设置[日志级别](https://kubernetes.io/docs/concepts/cluster-administration/logging/#cluster-level-logging-architectures)。推荐的设置方式是利用[fluentd](https://www.fluentd.org/)在外部储存(例如s3)上持久的收集日志。
```

Comment:
```suggestion
在大多数的情况下，这个命令没有办法获取所有的日志，因为调度程序的日志数量庞大，您需要设置[集群级别的日志收集](https://kubernetes.io/docs/concepts/cluster-administration/logging/#cluster-level-logging-architectures)。推荐的设置方式是利用[fluentd](https://www.fluentd.org/)在外部储存(例如s3)上持久的收集日志。
```

Reply:
Thanks for your review, I have update the context in new commit